mirdata/datasets/ikala.py

```diff
@@ -120,31 +120,54 @@ def __init__(self, track_id, data_home):
 
     @utils.cached_property
     def f0(self):
-        """F0Data: The human-annotated singing voice pitch"""
+        """The human-annotated singing voice pitch
+
+        Returns:
+            (F0Data): f0 """
         return load_f0(self.f0_path)
 
     @utils.cached_property
     def lyrics(self):
-        """LyricData: The human-annotated lyrics"""
+        """The human-annotated lyrics
+
+        Returns:
+            (LyricsData): LyricData
+        """
         return load_lyrics(self.lyrics_path)
 
     @property
     def vocal_audio(self):
-        """(np.ndarray, float): mono vocal audio signal, sample rate"""
+        """Vocal Audio ground truth
+
+        Returns:
+            (np.ndarray, float): audio signal, sample rate
+        """
         return load_vocal_audio(self.audio_path)
 
     @property
     def instrumental_audio(self):
-        """(np.ndarray, float): mono instrumental audio signal, sample rate"""
+        """Mono instrumental Audio ground truth
+
+        Returns:
+            (np.ndarray, float): audio signal, sample rate
+        """
         return load_instrumental_audio(self.audio_path)
 
     @property
     def mix_audio(self):
-        """(np.ndarray, float): mono mixture audio signal, sample rate"""
+        """Mono mixture Audio ground truth
+
+        Returns:
+            (np.ndarray, float): audio signal, sample rate
+        """
         return load_mix_audio(self.audio_path)
 
     def to_jams(self):
-        """Jams: the track's data in jams format"""
+        """the track's data in jams format:
+
+        Return:
+            (JAMS): return track data in jam format
```

Comment:
if we want the docs to link, we can put `jams.JAMS` for all of these.

Reply:
ok, i copy paste the numpy doc type

Reply:
done